app/main/controller/image_ocr_controller.py

```diff
@@ -37,9 +45,27 @@ def post(self):
         if not texts:
             return
 
+        #### calculate bounding boxes areas.
+        bounds = []
+        for page in response.full_text_annotation.pages:
+            for block in page.blocks:
+                    bounds.append(block.bounding_box)
+        total_area = 0
+        for annotation in bounds:
+            vertices = [(v.x, v.y) for v in annotation.vertices]
+            area = self.polygon_area(vertices)
+            total_area += area
+        image_width = response.full_text_annotation.pages[0].width
+        image_height = response.full_text_annotation.pages[0].height
+        image_area = image_width * image_height
+        percentage = (total_area / image_area) * 100
+
```

Comment:
Let's put all the new code on a function and warp the call to that function in a try... except block